src/MassTransit.Host/AssemblyScanner.cs

```diff
@@ -110,6 +111,19 @@ public Type GetHostBusFactoryType()
             }
         }
 
+        Assembly ReflectionOnlyLoadAssembly(string assemblyFile)

```

Comment:
I might change this to IEnumerable<Assembly> and use `SelectMany` above so that I can `yield break` instead of returning null;